theme/themes/pastanaga/elements/input.overrides

```diff
@@ -95,8 +95,19 @@ of an error is present, it overrides a default from SemanticUI grid definitions.
     cursor: pointer;
     text-align: initial;
 
-    &:focus {
-      outline: none;
+    &:focus,
```

Comment:
This change will apply the pink color also to other buttons such as the search icon from the header not just the x from the block chooser search.
![focus-hover-search-icon](https://github.com/plone/volto/assets/152852/ce35f344-e4ef-4f51-a28a-68567f9d784b)

Reply:
@ichim-david Ok, I made a modification to that CSS, now it uses only the add block search.